backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/appsettings.json

```diff
@@ -23,17 +23,31 @@
     "MetadataAccountName": "devstoreaccount1",
     "MetadataAccountKey": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==",
     "MetadataContainer": "metadata",
-    "MetadataBlobEndpoint": "http://127.0.0.1:10000/devstoreaccount1",
+    "MetadataBlobEndpoint": "https://pocaccessmanagement.blob.core.windows.net/",
     "DelegationsAccountName": "devstoreaccount1",
     "DelegationsAccountKey": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==",
     "DelegationsContainer": "delegationpolicies",
-    "DelegationsBlobEndpoint": "http://127.0.0.1:10000/devstoreaccount1",
-    "BlobLeaseTimeout": 15,
+    "DelegationsBlobEndpoint": "https://pocaccessmanagement.blob.core.windows.net/",
+    "ResourceRegistryAccountName": "pocaccessmanagement",
+    "ResourceRegistryAccountKey": "wiA0t9w4jYCn56f4MIfKLRf3Nm+Srg1d8W+9vNJhRPtJUVo+3Q+zC+uWCg3StPHb9QhIJoiu7qWU+ASt2gHQ0Q==",
```

Comment:
You should change your access key now :P
and reset the appsettings.json you commit to the default devstoreaccount1 as the others

Reply:
also for MetadataBlobEndpoint